src/OSPSuite.Infrastructure.Import/Services/IDataImporter.cs

```diff
@@ -17,15 +17,19 @@ public interface IDataImporter
       (IReadOnlyList<DataRepository> DataRepositories, ImporterConfiguration Configuration) ImportDataSets(
          IReadOnlyList<MetaDataCategory> metaDataCategories, 
          IReadOnlyList<ColumnInfo> columnInfos, 
-         DataImporterSettings dataImporterSettings
+         DataImporterSettings dataImporterSettings,
+         string moleculeName,
+         string molWeightName
       );
 
       IReadOnlyList<DataRepository> ImportFromConfiguration(
          ImporterConfiguration configuration,
          bool promptForConfirmation,
          IReadOnlyList<MetaDataCategory> metaDataCategories,
          IReadOnlyList<ColumnInfo> columnInfos,
-         DataImporterSettings dataImporterSettings
+         DataImporterSettings dataImporterSettings,
```

Comment:
I think it's time to create an configuration object where. with all those parameters aggregated together. We are repeating the same list of parameters twice already. Also I think it should be an cache<moleculeName, molweight> and not just a single value

Reply:
Done